modules/monitoring-exploring-the-visualized-metrics.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * monitoring/cluster-monitoring/examining-cluster-metrics.adoc
+
+[id="exploring-the-visualized-metrics_{context}"]
+= Exploring the visualized metrics
+
+After running the queries, the metrics are displayed on the interactive plot. The X axis of the plot represents time. The Y axis represents the metrics values. Each metric is shown as a colored graph. You can manipulate the plot and explore the metrics.
```

Comment:
I'd probably italicize *X* and *Y*, as first usage. From contributing guidelines: https://github.com/openshift/openshift-docs/blob/enterprise-4.1/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference

Reply:
It's not a term though. It just refers to something on the plot and says what it represents.
And it's also one of the many things I'm describing here, and not particularly important in any way. So I don't see a need to emphasize this. Besides, by that logic, I'd want to italicize "the interactive plot" and "colored graph". I think all that is unnecessary.
It's also used only once, so it's first and only usage.